nixos/modules/services/audio/gonic.nix

```diff
@@ -10,6 +10,10 @@ let
     mkKeyValue = lib.generators.mkKeyValueDefault { } " ";
     listsAsDuplicateKeys = true;
   };
+  assertKey = key: {
```

Comment:
Why not set a reasonable default in the module?

Reply:
Sometimes you don't think about everything in the first review and only realize something later on when the code is clearer. My notifications are also pretty clogged since the introduction of the merge queue as I get multiple notifications for PRs being merged.

The assertion is now correct used in the context of the module system and I just wanted to clarify if we have other options, but as it seems like it there are none we don't have a better alternative than to assert this if the program is not starting without it. That is okay, but I always think it is nice if that can be avoided, to avoid it.